app/models/user.rb

```diff
@@ -144,7 +144,7 @@ def self.create_blocked_scope(blocked)
   validates_uniqueness_of :login, :if => Proc.new { |user| !user.login.blank? }, :case_sensitive => false
   validates_uniqueness_of :mail, :allow_blank => true, :case_sensitive => false
   # Login must contain letters, numbers, underscores only
-  validates_format_of :login, :with => /\A[a-z0-9_\-@\.]*\z/i
+  validates_format_of :login, :with => /\A[a-z0-9_\-@\.\s]*\z/i
```

Comment:
@TribesTom I think we can ignore the Hound warnings here, as the original code didn't conform either (style issues have been fixed on `dev`, but not on the `release/4.0` branch).

`\s` doesn't strike me as the right choice here – we don't want to allow tabs or new line characters. AFAICT we only need to allow ` `.